Add doc for OCI installation (#1197)
jenkinsci/helm-charts: charts/jenkins/README.md

$ helm install [RELEASE_NAME] jenkins/jenkins [flags]
```

> Additionally, since version `5.6.0` the chart is available as OCI image on GitHub container registry and can be installed using the following command:

Reviewer: ```suggestion
Since version `5.6.0` the chart is available as an OCI image and can be installed using:
```

```console

> # Helm 3

Reviewer: ```suggestion
```
